www/nginx/src/opnsense/scripts/nginx/ngx_autoblock.php

```diff
@@ -0,0 +1,145 @@
+#!/usr/local/bin/php
+<?php
+
+/*
+ * Copyright (C) 2018 Fabian Franz
```

Comment:
thank you for using proper copyright layout here

Reply:
most of the time I am just copying it from my older plugins except some files which I am boilerplate using my rake task.

Reply:
I'm trying to make all of them one PHP style when I can:

```
/*
 * Copyright...
 *
 * [...]
 */
```